packages/docs/src/components/content/MaturityChecklist/MaturityChecklist.tsx

```diff
@@ -60,15 +60,15 @@ const MaturityChecklist = (props: MaturityChecklistProps) => (
       </MaturityChecklistItem>
     </ul>
 
-    <h3>Design</h3>
+    {/* <h3>Design</h3>
```

Comment:
Guessing this will be uncommented someday soon?

Reply:
Yep, I'm writing some tickets for next sprint for us to do the baseline design audit